src/components/Search/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react'
+
+import { app, header } from './index.module.css'
+
+import ClearButton from './ClearButton/ClearButton'
+import SearchButton from './SearchButton/SearchButton'
+import SearchInput from './SearchInput/SearchInput'
```

Comment:
maybe **SearchInputField** or **SearchField**?

Reply:
Could be, but what is wrong with SearchInput? 🙂

Reply:
Never met this before, sounded weird to me, but looks like it's used - let's keep, sorry for the noise :D